src/main/java/uk/gov/register/resources/RecordResource.java

```diff
@@ -45,7 +50,7 @@ public RecordResource(RegisterReadOnly register, ViewFactory viewFactory, Reques
     public RecordView getRecordByKey(@PathParam("record-key") String key) {
         httpServletResponseAdapter.addLinkHeader("version-history", String.format("/record/%s/entries", key));
 
-        return register.getRecord(key).map(viewFactory::getRecordMediaView)
+        return register.getDerivationRecord(key, IndexNames.RECORD).map(viewFactory::getRecordMediaView)
```

Comment:
I don't believe this change needs to be made. The `getRecord()` method in `PostgresRegister` maps down to `getDerivationRecord(key, IndexNames.RECORD)` as well.